src/oci-limits-mcp-server/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[project]
+name = "oci-limits-mcp-server"
```

Comment:
can you adjust the attributes in this file to match our template (https://github.com/oracle/mcp/blob/main/src/oci-api-mcp-server/pyproject.toml#L2)?

https://github.com/oracle/mcp/blob/main/src/oci-api-mcp-server/pyproject.toml